configuration/sitemaps.md

```diff
@@ -376,14 +382,14 @@ Image url="https://raw.githubusercontent.com/wiki/openhab/openhab/images/feature
 Video [item=<itemname>] [icon="<iconname>"] url="<url of video to embed>" [encoding="<video encoding>"]
 ```
 
-Allows to integrate a video presentation into a your Sitemap.
-Not all video encodings (formats) are supported, you may need to transcode your video.
-The video has to be reachable directly via URL.
-An embedded or protected video is not supported.
+Allows you to display a video as part of your Sitemap.
+Note: not all video encodings (formats) are supported; you may need to transcode your video.
+The video must be reachable directly via URL.
+An embedded and/or protected video are not supported.
 
-- `item` can refer to a String Item whose state is an URL to a video.    Some clients may not (yet) consider `item`.
-- `url` is the default URL from which to retrieve the video, if there is no associated Item or if the associated item's state is not an URL.
-- `encoding` can stay left empty for auto selection, for an MJPEG video please set the "mjpeg" encoding explicitly.
+- `item` can refer to a String Item whose state is a URL to a video. Some clients may not (yet) consider `item`.
+- `url` is the default URL from which to retrieve the video if there is no associated Item or if the associated item's state is not a URL.
+- `encoding` may be left empty for auto selection.  but please set it explicitly to "mjpeg" for an MJPEG video, or to "HLS" for an HTTP Live Streaming playlist (file with .m3u8 extension).
```

Comment:
"But"

I'd actually turn it around.

> "mjepg" for ..., "HLS" for ..., empty otherwise.

Reply:
This is what I put in my commit:

- `encoding` should be set to "mjpeg" for an MJPEG video, or "HLS" for an HTTP Live Streaming playlist (file with .m3u8 extension).  If you omit the `encoding` parameter, openHAB will attempt to automatically select the correct format.

Reply:
This comment has been addressed.